cmd/gocritic/main.go

```diff
@@ -62,7 +62,7 @@ func main() {
 	if cmd == nil {
 		terminate("unknown sub-command: "+sub, printSupportedSubs)
 	}
-	log.SetPrefix(sub + ": ")
+	log.SetPrefix("- ")
```

Comment:
Maybe remove prefix completely then?

Reply:
Just putting a `-` to make it more understandable that "Here is a new log!"

Reply:
Anyway we're expecting a log, so `-` doesn't provides useful context.